requirements.txt

```diff
@@ -1,5 +1,5 @@
 dawn==0.11.0
-django==2.2.24
+django~=2.2.13
```

Comment:
Not sure why this looks like it's getting downgraded

Reply:
requirements.txt now the same as the development branch